tools/dockerfile/grpc_artifact_python_manylinux_2_24_aarch64/Dockerfile

```diff
@@ -19,19 +19,19 @@
 # before https://github.com/dockcross/dockcross/pull/449
 # Thanks to that, wheel build with this image aren't actually
 # compliant with manylinux2014, but only with manylinux_2_24
-FROM dockcross/manylinux2014-aarch64:20200929-608e6ac
+FROM dockcross/manylinux2014-aarch64:20210826-19322ba
```

Comment:
It definitely sounds like updating the the image tag here would require updating the above comment as well.
Based on my knowledge, if you used a newer version of image than 20200929-608e6ac, you won't be able to build (precisely because of what's decribed in the comment).

Reply:
Ok, so the background is a bit complicated. 
- in the past, we had to use a version of dockcross/manylinux2014-aarch64 before https://github.com/dockcross/dockcross/pull/449, because newer versions didn't had a too old version of gcc and it was impossible to build grpc with them (and that had a side effect of non being able to generate manylinux 2_17 images, only manylinux 2_24). This is what the comment is about.
- just recently, in https://github.com/dockcross/dockcross/pull/562, the dockcross/manylinux2014-aarch64 image was improved to contain gcc9.3, which made the limitation from previous bulletpoint go away.

So, as a result:
- what you currently have works, just the comment "We use an older version of dockcross image..." is now out of date.
- the wheels generated by this image now are actually manylinux_2_17 compliant (not just manylinux_2_24). So in a followup PR, we could actually update the docker image name and some other details. But let's not do this here as what you currently have seems to be working just fine.